packages/gatsby-theme/src/components/Skip/Skip.stories.mdx

```diff
@@ -0,0 +1,55 @@
+import { Meta, Story, ArgsTable, Canvas } from "@storybook/addon-docs/blocks"
+import { Box } from "theme-ui"
+import { SkipLink, SkipContent } from "."
+
+<Meta
+  title='components/SkipLink'
+  component={SkipLink}
+  subcomponents={{
+    SkipContent,
+  }}
+/>
+
+# SkipLink
+
+Use this component to help jumps the user down to an anchor or target at the beginning of the main content
+
+## Props
+
+<ArgsTable of={SkipLink} />
+
+<Canvas>
+  <Story name='skiplink'>
+    <>
+      <SkipLink>Skip to main content</SkipLink>
```

Comment:
I don't know if there is something wrong on the story but I cant' see this `Skip to main content` link in the [storybook](https://deploy-preview-46--drupal-gatsby-theme.netlify.app/?path=/story/components-skiplink--skiplink)